Makefile

```diff
@@ -48,56 +60,48 @@ ENV HOME /home/$(USER)
 endef
```

Comment:
@deitch I think you left one copy of this block intact (and looking at your repot it indeed shows up there https://github.com/deitch/go-provision/blob/docker-build/Makefile#L52 ) 

Please take a look at how I rebased it on eve as well https://github.com/zededa/eve/pull/21

Reply:
Nice catch, thank you. Fixed.